test/models/comment_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CommentTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Lets add test cases for models or remove un-needed files if you feel there is no need to test case for any specific model ?